src/main/common/bitarray.h

```diff
@@ -15,6 +15,27 @@
  * along with Cleanflight.  If not, see <http://www.gnu.org/licenses/>.
  */
 
-bool bitArrayGet(const void *array, unsigned bit);
-void bitArraySet(void *array, unsigned bit);
-void bitArrayClr(void *array, unsigned bit);
+#include <stdbool.h>
+#include <stddef.h>
+
+/*
+ * These functions expect array to be 4-byte aligned since they alias array
+ * to an uint32_t pointer in order to work fast. Use the BITARRAY_DECLARE()
+ * macro to declare a bit array that can be safely used by them.
+ */
+
+typedef uint32_t bitarrayElement_t;
+
+#define BITARRAY_DECLARE(name, bits) bitarrayElement_t name[(bits + 31) / 32]
```

Comment:
You missed the alignment attribute.

Reply:
uint32_t should already be aligned to 4 bytes